### DISCUSSION TITLE
Polarium Gen 5

### DISCUSSION THREAD

**Discussion by cbbstewart** [Louisvdw/dbus-serialbattery#566] [New BMS requests]
Im looking for support of the Polarium Gen 5 Batteries. 

I can see from another thread someone has got it working. But im not sure what they added to the driver to make it work. 

Work was done by @arnoc-intuvision 

https://github.com/Louisvdw/dbus-serialbattery/issues/8#issuecomment-1164341379

Stewart

**Comment by Louisvdw** [comment]
They have not shared their change yet

**Comment by cbbstewart** [comment]
[POLARIUM Modbus Interface - Description (2).pdf](https://github.com/Louisvdw/dbus-serialbattery/files/9841008/POLARIUM.Modbus.Interface.-.Description.2.pdf) - i do have the modbus interface info now and it uses 115200 baud rate. I can talk to the battery using the polarium software.

**Reply by gegh1995** [reply]
I have a few Incell (older version, later Polarium) which use 9600 baud

**Comment by slarner** [comment]
Ive had a go at getting this to work still cant figure it out! ive got all the Register mappings from the document above. ive got about 20 of these batteries just sitting about which would be great to get to work with victron. Im also willing to pay for someones time to get it working! and access to one to figure it out! be great if @arnoc-intuvision shared what he did to make it work :)
